OmiseSwift/API Models/Capability.swift

```diff
@@ -389,6 +399,8 @@ extension Capability.Method {
                 self = .source(.billPayment(billPayment))
             case .barcode(let barcode):
                 self = .source(.barcode(barcode))
+            case .fpx(let fpx):
```

Comment:
What would you think about renaming the associated `fpx` everywhere to `fpxBank`? So like:
```swift
case .fpx(let fpxBank):
    self = .source(.fpx(fpxBank))
```
Since the `enum` is also named `FPXBank` it's maybe more clear?

Reply:
Yes, will rename them to fpxBank